modules/Rock.cmake

```diff
@@ -278,6 +278,20 @@ macro(rock_standard_layout)
         else()
             message(STATUS "unit tests disabled as ROCK_TEST_ENABLED is set to OFF")
         endif()
+
+    endif()
+    file(GLOB_RECURSE source_and_test_files CONFIGURE_DEPENDS "src/*[ch]pp" "src/*[ch]" "test/*[ch]pp" "test/*[ch]")
+
+    option(ROCK_CLANG_STYLING_CHECK_ENABLED 
+           "set to ON to styling check on test targets" OFF)
+    if (ROCK_CLANG_STYLING_CHECK_ENABLED)
+        rock_setup_styling_check(${PROJECT_NAME} "${source_and_test_files}")
```

Comment:
You will need to call enable_testing() explicitly (it is currently called only if the tests are enabled)